build-images.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-docker build --file 2.5/browsers/Dockerfile -t cimg/ruby:2.5.8-browsers  -t cimg/ruby:2.5-browsers .
-docker build --file 2.6/browsers/Dockerfile -t cimg/ruby:2.6.6-browsers  -t cimg/ruby:2.6-browsers .
-docker build --file 2.7/browsers/Dockerfile -t cimg/ruby:2.7.2-browsers  -t cimg/ruby:2.7-browsers .
```

Comment:
Curious - all other previous images will no longer be built as a result of this change, is this intended?

Reply:
Yes, this is intended. Once a SemVer tagged image is released, we don't typically re-release it unless something was seriously wrong with it. This highly improves the reliability of our images. Less moving parts.